### PR TITLE
Add display template interface

### PR DESCRIPTION
**Pull request by Nitwel**
I left the wrapper out for now and used the component directly but if you @rijkvanzanten prefer the use of an wrapper, I can definitely add that too.

### CODE REVIEW

File: app/src/interfaces/display-template/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		{
+			field: 'disabled',
+			type: 'boolean',
+			name: i18n.t('disabled'),
+			meta: {
+				width: 'half',
+				interface: 'toggle',
+				default_value: false,
+			},
+		},
```
Comment (rijkvanzanten) [inline app/src/interfaces/display-template/index.ts:30]: Value / disabled are props of every interface and aren't interface options

File: app/src/interfaces/display-template/index.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		{
+			field: 'collection',
+			name: i18n.t('collection'),
+			type: 'string',
+			meta: {
+				width: 'half',
+				interface: 'text-input',
+				default_value: '',
```
Comment (rijkvanzanten) [inline app/src/interfaces/display-template/index.ts:38]: Default value goes under `schema`, not `meta`

File: app/src/interfaces/display-template/display-template.vue [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<template>
+	<v-field-template />
```
Comment (rijkvanzanten) [inline app/src/interfaces/display-template/display-template.vue:2]: Don't forget to pass `collection` to the v-field-template component 🙂

**Comment by rijkvanzanten** [comment]
> prefer the use of an wrapper

No need for a wrapper element 👍

File: app/src/interfaces/display-template/display-template.vue (hunk 3 above)
Comment (Nitwel) [reply app/src/interfaces/display-template/display-template.vue:2]: No need when this wrapper is not used. 😄

**Comment by Nitwel** [comment]
This should now be ready to be merged.

**Pull request merged**